### PR TITLE
Document recent changes in image management

### PR DESCRIPTION
**Pull request by nadvornik**
# Description

This describes recent changes in image management.
Also, the screenshot for image list is improved.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by aaannz — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/administration/pages/image-management.adoc [hunk 1]
```diff
@@ -745,7 +745,7 @@ The generated pillar is available to all connected clients.
 
 
 Building an image requires several dependent steps.
-When the build fails, investigating Salt states results can help identify the source of the failure.
+When the build fails, investigating Salt states results and Build log can help identify the source of the failure.
```
Comment (keichwa) [inline modules/administration/pages/image-management.adoc:748]:
```suggestion
When the build fails, investigating Salt states results and build log can help identify the source of the failure.
```

File: modules/administration/pages/image-management.adoc [hunk 2]
```diff
@@ -768,19 +768,30 @@ The section contains some known issues when working with images.
...
 
 
-To list images available for building select menu:Images[Image List].
+To list available built images select menu:Images[Image List].
 A list of all images is displayed.
 
 image::images_list_images.png[scaledwidth=80%]
 
-Displayed data about images includes an image [guimenu]``Name``, its [guimenu]``Version`` and the build [guimenu]``Status``.
+Displayed data about images includes an image [guimenu]``Name``, its [guimenu]``Version``, [guimenu]``Revision`` and the build [guimenu]``Status``.
```
Comment (keichwa) [inline modules/administration/pages/image-management.adoc:779]:
```suggestion
Displayed data about images includes an image [guimenu]``Name``, its [guimenu]``Version``, [guimenu]``Revision``, and the build [guimenu]``Status``.
```

File: modules/administration/pages/image-management.adoc [hunk 3]
```diff
@@ -768,19 +768,30 @@ The section contains some known issues when working with images.
...
 You can also see the image update status with a listing of possible patch and package updates that are available for the image.
 
+For OS Images, the [guimenu]``Name`` and [guimenu]``Version`` fields originate from Kiwi sources and are updated at the end of successful build.
+During building or after failed build these fields show a temporary name based on profile name.
+
+[guimenu]``Revision`` is automatically increased after each successful build. For OS Images, multiple revisions can co-exist in the store.
+
+For Container Images the store holds only the latest revision. Information about previous revisions (packages, patches, etc.)
+are preserved and it is possible to list them with the [guimenu]``Show obsolete`` checkbox.
```
Comment (keichwa) [inline modules/administration/pages/image-management.adoc:788]:
```suggestion
For Container Images the store holds only the latest revision.
Information about previous revisions (packages, patches, etc.) are preserved and it is possible to list them with the [guimenu]``Show obsolete`` checkbox.
```

File: modules/administration/pages/image-management.adoc [hunk 4]
```diff
@@ -768,19 +768,30 @@ The section contains some known issues when working with images.
...
+
 Clicking the btn:[Details] button on an image provides a detailed view.
-The detailed view includes an exact list of relevant patches and a list of all packages installed within the image.
+The detailed view includes an exact list of relevant patches, list of all packages installed within the image and a build log.
+
+Clicking the btn:[Delete] button deletes the image from the list. It also deletes the associated pillar, files from OS Image Store
+and obsolete revisions.
```
Comment (keichwa) [inline modules/administration/pages/image-management.adoc:794]:
```suggestion
Clicking the btn:[Delete] button deletes the image from the list.
It also deletes the associated pillar, files from OS Image Store and obsolete revisions.
```

**Review by keichwa — APPROVED** [review]
Minor cosmetic proposals.
Within paragraphs, we usually want line-breaks at the end of sentences only.

**Pull request merged**